packages/admin-portal/src/components/ui/diff-media/diff-media.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import { cx } from 'linaria'
+import {
+  diffMediaContainer,
+  image,
+  greenBackground,
+  redBackground,
+  diffBlock,
+  arrow,
+  iconBlock,
+  iconImage,
+  mediaBlock,
+  mediaImage,
+} from './__styles__'
+
+export interface DiffMediaProps {
+  currentMedia?: string
+  changedMedia?: string
+  type?: string
+}
+
+export const DiffMedia = ({ currentMedia, changedMedia, type }: DiffMediaProps) => {
+  const blockStyle = type === 'icon' ? iconBlock : mediaBlock
+  const imageStyle = type === 'icon' ? iconImage : mediaImage
+  const isDiff = currentMedia !== changedMedia
+  return (
+    <div className={diffMediaContainer}>
+      <div className={cx(diffBlock, type === 'icon' ? iconBlock : mediaBlock, isDiff ? redBackground : '')}>
```

Comment:
I think you can change to avoid if else
`cx(diffBlock, type === 'icon' && iconBlock,  type !== 'icon' && mediaBlock, isDiff  && redBackground)`